server/src/main/java/org/apache/druid/segment/realtime/appenderator/AppenderatorImpl.java

```diff
@@ -691,6 +739,8 @@ public Object call() throws IOException
       if (sink.finishWriting()) {
         totalRows.addAndGet(-sink.getNumRows());
       }
+      // count hydrants for stats:
+      pushedHydrantsCount.addAndGet(IterableUtils.size(sink));
```

Comment:
You can use Iterables.size instead so you don't have to add new dependency on commons-collections4

Reply:
In next push